CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to abn
```

Comment:
I would put `abn` in "`" throughout the text.

Reply:
https://github.com/furrer-lab/abn/pull/82/commits/5a45776ad0f1c8dc384a0971dcdd08809099a07c